src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,56 @@
+# Navigate Between Application Windows
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Using the Dock
+
+The Dock will will designate a running application with a small white dot. Click the application to bring it's window into focus.
+
+![Dock Apps](/images/switching-apps/dock-apps.png)
+
+<!--
+View all windows in the current workspace by pressing `SUPER` + `D`. For more information about Workspaces, see [Using Workspaces](using-workspaces.md).
+
+![Display All Windows](/images/locate-launch-applications/display-all-windows.png)
+
+Use the the `🠐` `🠑` `🠒` `🠓` keys or mouse to move between windows. Select a window with left-click or by pressing `Enter`.
+
+![Select Displayed Window](/images/locate-launch-applications/select-displayed-window.png)
+
+Switch back to the previously selected window by pressing `Alt` + `Esc`.
+-->
+
+## Using the Launcher
+
+Pressing `SUPER` will display the Launcher. All currently running applications will be listed below the search bar. Select a running application with the `🠑` `🠓` keys, and hit `Enter` to bring the window into focus.
+
+![Launcher Running Apps](/images/switching-apps/launcher-running-apps.png)
+
+## Using Keyboard Shortcuts
+
+Pressing `SUPER` + `Tab` or `Alt` + `Tab` will display a list of running application icons across the screen. Continue holding `SUPER` or `Alt` and pressing `Tab` to toggle between applications. You can also navigate between applications with the `🠐` `🠒`  keys.
+
+![Toggle Apps](/images/switching-apps/toggle-apps.png)
+
+Mouse over or release `Tab` on highlighted application icons to display multiple open windows for that application. Continue holding `SUPER` or `Alt` and press ` to toggle between the multiple windows.
```

Comment:
An alternative to using the tilde is pressing the down arrow key while the app is selected-- that brings the selection down into the window list so the left/right arrow keys can be used.